WordPress/Classes/ViewRelated/Post/PostCardTableViewCell.m

```diff
@@ -359,20 +359,15 @@ - (void)configureMetaButtons
     [self resetMetaButton:self.metaButtonRight];
     [self resetMetaButton:self.metaButtonLeft];
 
-    // We don't have comment and like counts for self-hosted sites.
-    if (![self.contentProvider isWPcom]) {
-        return;
-    }
-
     NSMutableArray *mButtons = [NSMutableArray arrayWithObjects:self.metaButtonLeft, self.metaButtonRight, nil];
-    if ([self.contentProvider numberOfComments] > 0) {
+    if ([self.contentProvider supportsComments] && [self.contentProvider numberOfComments] > 0) {
```

Comment:
Not sure what's gained by adding the `supportsComments` and `supportsLikes` checks since the count would just be 0 if not supported.